### PR TITLE
Release/0.9.0

### PR DESCRIPTION
**Pull request by adatzer**

### CODE REVIEW

File: .github/workflows/cd.yml [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+      uses: softprops/action-gh-release@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        tag_name: ${{ github.ref }}
+        name: Snowplow Python Tracker v${{ steps.version.outputs.PYTHON_TRACKER_VERSION }}
+        draft: false
+        prerelease: ${{ contains(steps.version.outputs.TAG_VERSION, 'rc') }}
+
+  publish:
```
Comment (paulboocock) [inline .github/workflows/cd.yml:79]: Should we publish before we release to Github instead? I think the publish step is more likely to fail but we'd have released to Github already once it had failed.
If we do this, we should also move the Version check (perhaps even before we build it, or to the beginning of publish if thats preferred?)

File: CHANGES.txt [hunk 2]
```diff
@@ -1,3 +1,26 @@
+Version 0.9.0 (2021-04-20)
```
Comment (paulboocock) [inline CHANGES.txt:1]: Just a reminder to update this date on release

File: LICENSE [hunk 3]
```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2013-2021 Snowplow Analytics Ltd.
```
Comment (paulboocock) [inline LICENSE:190]: 👍

File: README.rst [hunk 4]
```diff
@@ -1,19 +1,30 @@
...
-Python Analytics for Snowplow 
+Python Analytics for Snowplow
 ======================================================
 .. image:: https://img.shields.io/static/v1?style=flat&label=Snowplow&message=Early%20Release&color=014477&labelColor=9ba0aa&logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAMAAAAoLQ9TAAAAeFBMVEVMaXGXANeYANeXANZbAJmXANeUANSQAM+XANeMAMpaAJhZAJeZANiXANaXANaOAM2WANVnAKWXANZ9ALtmAKVaAJmXANZaAJlXAJZdAJxaAJlZAJdbAJlbAJmQAM+UANKZANhhAJ+EAL+BAL9oAKZnAKVjAKF1ALNBd8J1AAAAKHRSTlMAa1hWXyteBTQJIEwRgUh2JjJon21wcBgNfmc+JlOBQjwezWF2l5dXzkW3/wAAAHpJREFUeNokhQOCA1EAxTL85hi7dXv/E5YPCYBq5DeN4pcqV1XbtW/xTVMIMAZE0cBHEaZhBmIQwCFofeprPUHqjmD/+7peztd62dWQRkvrQayXkn01f/gWp2CrxfjY7rcZ5V7DEMDQgmEozFpZqLUYDsNwOqbnMLwPAJEwCopZxKttAAAAAElFTkSuQmCC
     :alt: Early Release
-    :target: https://github.com/snowplow/snowplow/wiki/Tracker-Maintenance-Classification
-.. image:: https://travis-ci.org/snowplow/snowplow-python-tracker.svg?branch=master
+    :target: https://docs.snowplowanalytics.com/docs/collecting-data/collecting-from-own-applications/tracker-maintenance-classification/
+.. image:: https://img.shields.io/github/workflow/status/snowplow/snowplow-python-tracker/build
```
Comment (paulboocock) [inline README.rst:7]: Whilst we usually use shields.io for badges, I think preferring GitHubs own badge for builds is better: https://github.com/snowplow/snowplow-python-tracker/actions/workflows/ci.yml/badge.svg

File: README.rst [hunk 5]
```diff
@@ -28,26 +39,24 @@ With this tracker you can collect event data from your Python-based applications
...
 
-.. _`Technical Docs`: https://github.com/snowplow/snowplow/wiki/Python-Tracker
-.. _`Setup Guide`: https://github.com/snowplow/snowplow/wiki/Python-Tracker-Setup
-.. _`Roadmap`: https://github.com/snowplow/snowplow/wiki/Python-Tracker-Roadmap
-.. _`Contributing`: https://github.com/snowplow/snowplow/wiki/Python-Tracker-Contributing
+.. _`Technical Docs`: https://docs.snowplowanalytics.com/docs/collecting-data/collecting-from-own-applications/python-tracker/
+.. _`Setup Guide`: https://docs.snowplowanalytics.com/docs/collecting-data/collecting-from-own-applications/python-tracker/setup/
+.. _`Contributing`: https://github.com/snowplow/snowplow-python-tracker/blob/master/CONTRIBUTING.md
 
 Quickstart
```
Comment (paulboocock) [inline README.rst:61]: I've recently started updating this title to `Maintainer Quickstart` to remove any confusion that you need to do this to use the tracker.

**Review by paulboocock — APPROVED** [review]
LGTM! Nice release, that Github Actions are particularly nice!

This is likely already on your radar but upon release, we should ensure the documentation appropriately covers how to use this with redis or celery.

File: .github/workflows/cd.yml (hunk 1 above)
Comment (adatzer) [reply .github/workflows/cd.yml:79]: Thank you for pointing this out. Indeed, the release is less likely to fail. And the Version-check also makes sense to move ahead to fail fast if there is mismatch.

**Review by paulboocock — APPROVED** [review]
LGTM!

**Pull request merged**